### PR TITLE
Updating private law shutter name

### PR DESCRIPTION
**Pull request by AlokDatta**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -96,8 +96,8 @@ parameters:
       - name: 'rpts'
         shutter: 'default'
         subscription: DCD-CFTAPPS-PROD
-      - name: 'privatelaw'
-        shutter: 'privatelaw'
+      - name: 'privatel'
```
Comment (thomast1906) [inline azure-pipelines.yml:99]: Is current not more meaningful? Curious on why the change?

Comment (AlokDatta) [reply azure-pipelines.yml:99]: Hi @thomast1906 , 

Definitely I ageree, but I have been trying to figure out why the shuttering is not working. I think the only reason is this when I go through the hmcts.github.io -> **Use the first 8 characters only, of the name attribute from the env.tfvar**

Reference: https://hmcts.github.io/cloud-native-platform/path-to-live/shutter.html#uploading-static-shutter-pages-to-azure-storage-account
Use the first 8 characters only, of the name attribute from the env.tfvar file e.g. If the name attribute in env.tfvars file is xui-manage-org then ${shuttername} will be xuimanag.

Comment (thomast1906) [reply azure-pipelines.yml:99]: ok - thanks for that

**Review by thomast1906 — APPROVED** [review]
